pkg/storageincentives/agent.go

```diff
@@ -426,6 +418,8 @@ func (a *Agent) handleSample(ctx context.Context, round uint64) (bool, error) {
 		return false, err
 	}
 
+	a.logger.Info("produced sample", "hash", sample.ReserveSampleHash, "radius", sample.StorageRadius)
```

Comment:
Would it be better if it also prints the round no? That way we can aggregate the logs.